### PR TITLE
[Console] Add doc about lockableTrait

### LINKED ISSUE #19688: [Console] Add a way to use custom lock factory in lockableTrait
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#54135
| PR author(s) | @VincentLanglet
| Merged in    | 7.1

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 7.1 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by VincentLanglet**
Closes https://github.com/symfony/symfony-docs/issues/19688

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Why do I get `[Missing class] Class, interface or trait with name "Symfony\Component\Lock\LockFactory" does not exist` ?

File: console/lockable_trait.rst [hunk 1]
```diff
@@ -43,4 +43,23 @@ that adds two convenient methods to lock and release commands::
         }
     }
 
+By default, the LockableTrait will use the ``FlockStore``, or the ``SemaphoreStore`` if available,
```
Comment (javiereguiluz) [inline console/lockable_trait.rst]: This part is not fully clear to me:

```rst
will use the ``FlockStore``, or the ``SemaphoreStore`` if available [...]
```

I propose to reword it to better explain which of the two stores will be tried first:

```rst
The LockableTrait will use the ``SemaphoreStore`` if available and will default
to ``FlockStore`` otherwise. You can override this behavior [...]
```

**Comment by javiereguiluz** [comment]
Vincent, you can ignore the code block error. It's a false positive that will be fixed when we update the code in that linter. Thanks.

File: console/lockable_trait.rst (hunk 1 above)
Comment (VincentLanglet) [reply console/lockable_trait.rst]: Thanks for the feedback, it's changed.

**Review by javiereguiluz — APPROVED** [review]
